data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/deserializers/NetworkActionDeserializer.java

```diff
@@ -124,6 +126,12 @@ private static Topology deserializeTopology(JsonParser jsonParser, SimpleCrac si
                     actionType = jsonParser.readValueAs(ActionType.class);
                     break;
 
+                case EXTENSIONS:
```

Comment:
I know your dev isn't really the one that caused it, but I feel the
while (!jsonParser.nextToken().isStructEnd()) { 
    switch (jsonParser.getCurrentName()) {\
        ...
    }
}
code parts could be refactored to avoid duplication.

Reply:
As discussed, I did some modifications to reduce code duplication in this file. If we want to refactor further, we will do it in a separate pull request.